Scoop/Controller/Onboarding/OnboardingPageViewController.swift

```diff
@@ -93,6 +93,17 @@ extension OnboardingPageViewController: UIPageViewControllerDataSource {
         return index
     }
     
+    func setupNextButton(view: UIView, action: UIAction, button: UIButton) {
```

Comment:
Remove the UIView, UIAction, and UIButton from the parameters and instead set them directly within the function `setupNextButton`

Reply:
I removed the UIView and UIButton from the parameters, but since the UIAction is different for each view controller, I'm not sure how I would be able to include the UIAction definition within the function.